scripts/run_spark_connect_server.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/bash
+
+# This script was inspired by https://github.com/pyspark-ai/pyspark-ai/blob/master/run_spark_connect.sh
+
+# The Hadoop and Spark versions are set as environment variables for this script.
+echo "The HADOOP_VERSION is $HADOOP_VERSION"
```

Comment:
It seems that Hadoop 2 [is EOL already](https://www.versio.io/product-release-end-of-life-eol-Hadoop-Apache.html). Do we really need to test Quinn against it starting from 1.0?

Reply:
Don't think so.

Shall I:
- Remove the version assignment in the shell script & just hardcode it here?
- And, remove setting the HADOOP_VERSION as an environment variable in the `ci.yml`?

Reply:
Yes. Let's fix HADOOP_VERSION to `3` / `hadoop3`?